.github/workflows/deploy-program.yml

```
@@ -41,40 +51,56 @@ jobs:
     name: Programs
     uses: ./.github/workflows/build-programs.yml
     secrets: inherit
+    with:
+      git_ref: ${{ inputs.git_ref }}
```

actionlint (1.7.12)

[error] input "git_ref" is not defined in "./.github/workflows/build-programs.yml" reusable workflow. defined inputs are "rust", "solana"

(workflow-call)